data/hai ships.txt

```diff
@@ -45,7 +45,7 @@ ship "Aphid"
 	explode "tiny explosion" 10
 	explode "small explosion" 20
 	explode "medium explosion" 15
-	description "The Aphid is a light freighter designed by the Hai. It has only minimal armament."
+	description "The Aphid is the lightest ship designed by the Hai. Similar to the Human Star Barge, it carries only a single turret in case of emergencies, and has excellent cargo capacity for it's size."
```

Comment:
Human is never capitalized in game.
`it carries only a single turret in case of emergencies` -> `it can carry only a single turret for defensein case of emergencies`?